### PR TITLE
Fixes #191 - Reworked keyboard shortcuts

### LINKED ISSUE #191: Rework advanced > keyboard shortcuts

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

File: images/advanced/profile-shortcuts.png
Comment (mgruner) [inline images/advanced/profile-shortcuts.png:1]: Therer is an issue here: the Layout Reference is only shown in the `development` Rails environment. Regular users on production won't see it.

File: advanced/keyboard-shortcuts.rst [hunk 1]
```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
...
-   .. figure:: /images/extras/profile-and-settings/profile-and-settings.png
+
+   .. figure:: /images/advanced/profile-shortcuts.png
       :alt: User submenu
       :align: center
       :scale: 85%
- 
+
    Alternately, bring it up with one of the shortcuts below
    (shortcut-ception!)
```
Comment (mgruner) [inline advanced/keyboard-shortcuts.rst:20]: What does `shortcut-ception` mean? I'm not familiar with this term.

File: advanced/keyboard-shortcuts.rst [hunk 2]
```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
...
 
-:As-you-type:
+Navigation
+^^^^^^^^^^
 
-   * Press ``Cmd`` + ``I`` to enter Italics mode,
+====================================  ===================================================
+Key / key combination                       Function
+====================================  ===================================================
+``shift`` + ``ctrl`` + ``d``          Show Dashboard
```
Comment (mgruner) [inline advanced/keyboard-shortcuts.rst:45]: dashboard

File: advanced/keyboard-shortcuts.rst [hunk 3]
```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
...
+``shift`` + ``ctrl`` + ``o``          Show Overviews
```
Comment (mgruner) [inline advanced/keyboard-shortcuts.rst:46]: overviews

File: advanced/keyboard-shortcuts.rst [hunk 4]
```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
...
+``shift`` + ``ctrl`` + ``e``          Log out
+``shift`` + ``ctrl`` + ``h``          Show list of shortcuts
+``shift`` + ``ctrl`` + ``w``          Close current tab
+``shift`` + ``ctrl`` + ``tab``        Show next tab
+``shift`` + ``ctrl`` + ``shift+tab``  Show previous tab
+``shift`` + ``ctrl`` + ``return``     Confirm/submit in dialogs
+``↑`` / ``↓``                         Move selection/cursor up and down
+``←`` / ``→``                         Move selection/cursor left and right
+``enter``                             Select item / confirm
+``shift`` + ``ctrl`` + ``.``          Copy current object number (e.g. Ticket number)
```
Comment (mgruner) [inline advanced/keyboard-shortcuts.rst:59]: ticket

File: advanced/keyboard-shortcuts.rst [hunk 5]
```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
...
+``shift`` + ``ctrl`` + 2x ``.``       Add the title of the object to the number
+``shift`` + ``ctrl`` + 3x ``.``       Add the object link URL to the number and title
+====================================  ===================================================
+
+
+Translations
+^^^^^^^^^^^^
+Note: you need to have admin permissions to enable it.
```
Comment (mgruner) [inline advanced/keyboard-shortcuts.rst:67]: to use this. (?)

File: advanced/keyboard-shortcuts.rst [hunk 6]
```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
...
+.. figure:: /images/advanced/keyboard-shortcuts.png
+   :alt: Keyboard shortcut cheat sheet
+   :align: center
+   :scale: 85%
+
+   The keyboard shortcut cheat sheet.
+
 
-Keyboard shortcuts can be used to apply rich-text formatting in one of two ways:
+List of available keyboard shortcuts
```
Comment (mgruner) [inline advanced/keyboard-shortcuts.rst:36]: Title case?

File: advanced/keyboard-shortcuts.rst [hunk 7]
```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
...
+``shift`` + ``ctrl`` + ``g``          Reply to the last article
+``shift`` + ``ctrl`` + ``j``          Switch the visibility of the article between internal and public
+``shift`` + ``ctrl`` + ``c``          Set state of the ticket to "closed"
+``shift`` + ``ctrl`` + ``←`` / ``→``  Navigate through article
+``::``                                Insert text module (while composing an article)
+``??``                                Insert knowledge base article (while composing an article)
+``@@``                                Mention a user (while composing an article)
+====================================  ===================================================================
+
+Text editing
```
Comment (mgruner) [inline advanced/keyboard-shortcuts.rst:101]: Title case?

File: advanced/keyboard-shortcuts.rst [hunk 8]
```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
...
+^^^^^^^^^^^^
+
+.. tip::
+   You can apply a text format *before* typing or *after* typing.
+
+   Before typing:
+
+   * Press ``cmd`` + ``i`` to enter Italics mode,
```
Comment (mgruner) [inline advanced/keyboard-shortcuts.rst:109]: italics. You can highlight it - if needed - with italic formatting, just as 'normal text mode' below?

**Review by mgruner — APPROVED** [review]
@ralf401 thanks for the nice improvement. Just some small style suggestions. Hit me up if something is unclear.

File: advanced/keyboard-shortcuts.rst (hunk 1 above)
Comment (ralf401) [reply advanced/keyboard-shortcuts.rst:20]: Me neither. I search for it and it probably refers to a recursion (open shortcuts with a shortcut). I have changed it now. Probably not that much used then.

**Pull request merged**